.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "github action updates"
+
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    commit-message:
+      prefix: "pip updates"
```

Comment:
Please note that pymkv2 uses Poetry for dependency management, so using pip updates might not help you get the latest updates.

Reply:
Yeah.  I was looking at the documentation for [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file) and it says to indicate pip even for Poetry.  I'm using this in another one of my repos.

The biggest pain I had with it was it creating multiple PRs to upgrade the dependencies.  I believe there is a way to group them though.  I am also exploring in one of my repos to auto merge if from dependabot.

Reply:
Updated the PR to also add dependabotautomerge.yml to automatically merge the dependabot changes (provided the build succeeds on the other checks)